src/i_sound.c

```diff
@@ -775,7 +789,14 @@ void I_BindSoundVariables(void)
         "Variable pitch bend range (100 = None)");
     BIND_BOOL_SFX(full_sounds, false, "Play sounds in full length (prevents cutoffs)");
     BIND_NUM_SFX(snd_channels, MAX_CHANNELS, 1, MAX_CHANNELS,
-        "Maximum number of simultaneous sound effects");
+        "Number of sound channels");
+    BIND_BOOL_SFX(snd_limiter, false, "Use sound output limiter");
+    BIND_NUM(snd_channels_per_sfx, 5, 0, MAX_CHANNELS,
+        "[Limiter] Max number of channels allowed to simultaneously play the "
+        "same sound (0 = Off)");
+    BIND_NUM(snd_volume_per_sfx, 5 * 100, 0, MAX_CHANNELS * 100,
+        "[Limiter] Max volume allowed for a sound that is played "
+        "simultaneously by multiple channels [percent] (0 = Off)");
```

Comment:
@jackrjli Thanks for the feedback! I increased the range and set the default `snd_volume_per_sfx` to `500`. The result should be more subtle and only catch extreme cases (like Stickney E2M2). Also, a value of `0` (Off) can now be used for either setting.